client/src/components/Landing/Landing.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+import NavBar from '../NavBar/NavBar'
+import FirstSection from './FirstSection'
+import Footer from './Footer'
+import GetStartedModal from './GetStartedModal'
```

Comment:
I left a comment earlier on the code for this component, I'm just wondering why we don't use the 'SignUpModal' component we've already built.